iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/list.inl

```diff
@@ -529,7 +529,7 @@ inline bool list<T, Capacity>::IteratorBase<IsConstIterator>::operator!=(
 template <typename T, uint64_t Capacity>
 template <bool IsConstIterator>
 inline typename list<T, Capacity>::template IteratorBase<IsConstIterator>::reference
-    list<T, Capacity>::IteratorBase<IsConstIterator>::operator*() const noexcept
+list<T, Capacity>::IteratorBase<IsConstIterator>::operator*() const noexcept
```

Comment:
Oh, here is a very important `inline` missing. Could you please add this otherwise someone will get sooner or later a double definition error during linking.

Reply:
Please have a look two lines above ;)